projects/kubernetes/autoscaler/1-20/GIT_TAG

```diff
@@ -0,0 +1 @@
+cluster-autoscaler-1.20.3
```

Comment:
This may open a can of worms that isnt worth it, but packages doesnt support 1-20. we may be able to get away just not building 1.20 for this one?